status/k8s.go

```diff
@@ -178,7 +178,7 @@ func (k *K8sStatusCollector) podCount(ctx context.Context, status *Status) error
 
 	if pods != nil && len(pods.Items) != 0 {
 		for _, pod := range pods.Items {
-			if !pod.Spec.HostNetwork {
+			if !pod.Spec.HostNetwork && pod.Status.Phase != corev1.PodSucceeded {
```

Comment:
I wonder whether we should exclude all but running pods. IMO including e.g. failed and pending pods doesn't make much sense as far as Cilium is concerned.

So how about:

```suggestion
			if !pod.Spec.HostNetwork && pod.Status.Phase == corev1.PodRunning` {
```

which would also exclude completed pods (among others)?

Reply:
@tklauser Yes, I agree with you. Commiting your suggestion, thanks!

Reply:
Thanks @felfa01. Could you please squash the two commits into one (i.e. dropping "my" commit and folding it into your change) and force-push the branch?

Reply:
@tklauser Sure thing. Should be fine now.